tests/helpers/stubs.py

```diff
@@ -335,6 +349,44 @@ def isActive(self):
         return self._started
 
 
+class FakeConfigSection:
+
+    """A stub for a KeyValue entry in configdata.DATA."""
+
+    def __init__(self, *entries):
+        self.values = collections.OrderedDict()
+        self.descriptions = {}
+        for name, value, desc in entries:
+            self.values[name] = value
+            self.descriptions[name] = desc
+
+    def __iter__(self):
+        """Iterate over all set values."""
+        return self.values.__iter__()
```

Comment:
nitpick: This should be `iter(self.values)` (just like you wouldn't do `self.values.__getitem__(key)` below) :wink: